rider/urls.py

```diff
@@ -9,16 +9,16 @@
                        url(r'^register_push/$',
                            views.RiderPushUpdateAPI.as_view()),
 					# get affinity group data
-					url(r'^list_group/(?P<r_id>\d+)/$', 
+					url(r'^list_group/(?P<r_id>[^/]+)/$', 
```

Comment:
Sorry about that. It's a habit I have that makes paths slightly more restrictive, but it really doesn't matter and would likely break mobile if there is a case where the call is made without the trailing slash. Thanks for the catch :)